addon/webextension/content-script.js

```diff
@@ -54,11 +59,18 @@ function findAndReplace(wordList) {
     <br/><a href="${SUPPORTURL}" target="_blank", rel="noopener noreferrer">
```

Comment:
nit: There shouldn't be a rogue comma in the middle of the `<a>` tag.

Reply:
good catch!